src/main/webapp/query-control.js

```diff
@@ -45,23 +45,22 @@ function passQuery() {
   const location = document.querySelector(
     '#location option[value=\'' + locationInput + '\']').dataset.value;
 
-  const actionToText = new Map();
-  actionToText['moved'] = 'moved to';
-  if (actionToText.has(action)) {
-    action = actionToText[action];
-  }
-  const description = {action: action, age: personType};
+  const actionToPerson = new Map();
+  actionToPerson.set(
+        'live', 'Population',
+      ).set(
+        'work', 'Workers',
+      ).set(
+        'moved', 'New inhabitants',
+      );
+  const description = `${actionToPerson.get(action)} (${personType.replace('-', ' ')})`;
```

Comment:
For some reason I don't love the current look with the parentheses, I think I'd prefer it to be like "personType action" ie. Male Workers, New Male Inhabitants, but this would probably not translate well for some queries. Open to thoughts

Reply:
Yeah, I thought about this a lot. Male workers etc. would be way better, but then it doesn't work for age... Do you think it's worth hardcoding in two different description types based on the personType?

Reply:
Yeah I guess the way it is now works better since it's more dynamic. Switching it would overly complicate things, and the descriptions are currently clear to a user